lib/api/openapi/documents/definitions.yaml

```diff
@@ -0,0 +1,32 @@
+definitions:
```

Comment:
Those definitions should  not be in the `documents` directory since they are generic for all API actions.  
Proposal: `openapi/payloads.yaml` to match the documentation
![image](https://user-images.githubusercontent.com/4447392/143558448-a50df8ef-dcc6-4d44-b00b-59c5bcac3d32.png)